p2panda-net/src/network.rs

```diff
@@ -657,10 +657,8 @@ mod sync_protocols {
             }
 
             sink.flush().await?;
-            sink.close().await?;
```

Comment:
This turned out to be the cause of at least some of the closed connection errors.